task1/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>課題1</title>
+  </head>
+  <body>
+    <div>
+      <ul id="js_list"></ul>
+    </div>
+
+    <script>
+      const js_list = document.getElementById('js_list');
+      //取得したい要素が決まっているのでHTMLにIDを付与しgetElementByIdで取得
```

Comment:
変数名に`js`は要らないです。取得してきた要素が`li`であればわかりますが、`ul`に対して`list`という名前は適切でないです。ここに関しては
```suggestion
      const ul = document.getElementById('js_ul');
```
 ではないでしょうか。

Reply:
ご指摘いただいたことで変な記述になっていることに気がつきました。
ulに対する書き方に修正しました。
ありがとうございます！！